ci/README.md

```diff
@@ -110,3 +110,22 @@ Note that you can use the `dir` parameter in `run` to define the working directo
 ```
 
 Don't forget to remove separate pipelines that were created for testing.
+
+### Versioning Guide
+
+For creating a new release please follow the versioning guide based on the [Semantic Versioning Specification](https://semver.org/).
+
+* **Major Version** (*X*.y.z) -- incremented if any backwards incompatible changes are introduced to the public API
+
+  Used for new HAProxy **minor** versions, as they could contain breaking changes.
+* **Minor Version** (x.*Y*.z) -- incremented if new, backwards compatible functionality is introduced to the public API
+  
+  Used when upgrading dependencies (e.g. PCRE, socat, etc.) or HAProxy **patch** versions.
+* **Patch Version** (x.y.*Z*) -- incremented if only backwards compatible bug fixes are introduced)
+  
+  Used for documentation updates, changes in the test suite or any updates in the testing frameworks (e.g. ginkgo).
+
+The `haproxy-boshrelease` also contains patches (see [haproxy-patches](../haproxy-patches)). The patched version is denoted by appending a hyphen and key word `patched`, like `11.17.4-patched`.
```

Comment:
This is in conflict with the SemVer spec which uses hyphens (`-`) for pre-releases

Reply:
In theory the `patched` notion could also be considered build metadata, but I'm not sure if this is confusing and it might be easier to just denote this by the name of the binary.

Reply:
I would suggest `11.17.4+2.6.9-patched`. It is actually the HAProxy that is patched and `-` is valid in the build metadata.

Reply:
Applied